src/collectors/memory/memory.py

```diff
@@ -59,7 +60,9 @@ def get_default_config(self):
         """
         config = super(MemoryCollector, self).get_default_config()
         config.update({
-            'path':     'memory',
+            'path': 'memory',
+            'method': 'Threaded',
+            'force_psutil': 'False'
```

Comment:
If this is true and `psutil` is false, we should probably at least log a warning or something.